config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -997,12 +529,8 @@ spec:
                 cruiseControlEndpoint:
                   type: string
                 cruiseControlTaskSpec:
-                  description: CruiseControlTaskSpec specifies the configuration of
-                    the CC Tasks
                   properties:
                     RetryDurationMinutes:
-                      description: RetryDurationMinutes describes the amount of time
```

Comment:
is the removal of description fields on-purpose or a side effect of version upgrade?

Reply:
Good catch, I will check this this is a side effect of upgrading to the 0.2.5 generator tool. Will check.

Reply:
I found the problem will fix it on the next commit.

Reply:
awesome - thanks Balint!